fl4health/clients/basic_client.py

```diff
@@ -689,6 +734,8 @@ def setup_client(self, config: Config) -> None:
         train_loader, val_loader = self.get_data_loaders(config)
         self.train_loader = train_loader
         self.val_loader = val_loader
+        test_loader = self.get_test_data_loader(config)
```

Comment:
Any reason to do this in two stages instead of just doing `self.test_loader = ...`?

Reply:
fixed!